components/authentication/auth.js

```diff
@@ -0,0 +1,19 @@
+import jwt from 'jsonwebtoken'
+import User from '../users/user'
+
+const auth = async (req, res, next) => {
+  try {
+    const token = req.header('Authorization').replace('Bearer ', '')
+    const decoded = jwt.decode(token, process.env.SECRET_KEY)
+    const user = await User.findOne({ _id: decoded._id, 'tokens.token': token })
```

Comment:
Can you explain this of `tokens.token`?

Reply:
Fixed

Reply:
We allowed users to have multiples tokens for cases when they signing in different devices.